modules/manage/pages/kubernetes/storage/delete-persistentvolume.adoc

```diff
@@ -20,13 +21,27 @@ To delete a PersistentVolume, follow these steps to ensure that your data is mov
 kubectl get persistentvolume
 ----
 
-. Decommission the broker that has a PersistentVolumeClaim (PVC) bound to the PV. Decommissioning ensures no data loss by gracefully moving the broker's topic partitions and replicas to other brokers in the cluster. See xref:./decommission-brokers.adoc[Decommission Brokers].
+. xref:manage:kubernetes/decommission-brokers.adoc[Decommission the broker] that has a PersistentVolumeClaim (PVC) bound to the PV.
++
+Decommissioning ensures no data loss by gracefully moving the broker's topic partitions and replicas to other brokers in the cluster.
+
 . Delete the PVC that is bound to your PV:
 +
 [,bash]
 ----
 kubectl delete persistentvolumeclaim <pvc-name> --namespace <namespace>
 ----
++
+[NOTE]
+====
+To prevent accidental loss of data, PersistentVolumesClaims are not deleted when Redpanda brokers are removed from a cluster.
+It is your responsibility to delete PersistentVolumeClaims when they are no longer needed.
```

Comment:
Redpanda strongly recommends that you delete PersistentVolumeClaims when they are no longer needed.